### PR TITLE
[DO NOT MERGE] Internal command

### PR DESCRIPTION
**Pull request by krystian-hebel**
Opened a PR for easier comparison of changes, but this should be left as a different branch.

### CODE REVIEW

**Comment by miczyg1** [comment]
@krystian-hebel wouldn't be the diff better when comparing against `custom_shell_pkg`?

**Comment by krystian-hebel** [comment]
@miczyg1 I don't think so. Most of these changes were undone, except for the ones in `PlatformPkg.fdf`.

File: Vlv2TbltDevicePkg/PlatformPkg.dec [hunk 1]
```diff
@@ -239,4 +243,4 @@
   gEfiEsrtTableGuid              = { 0xb122a263, 0x3661, 0x4f68, { 0x99, 0x29, 0x78, 0xf8, 0xb0, 0xd6, 0x21, 0x80 } }
   gEfiCapsuleCrashLogVarGuid     = { 0xf3ff1468, 0x04ba, 0x4966, { 0x9f, 0xb2, 0xe4, 0xa7, 0x90, 0x05, 0x46, 0x50 } }
   gEfiCapsuleCrashGuid           = { 0x0e1d2972, 0x65af, 0x4ac1, { 0xbf, 0xa3, 0xce, 0xf4, 0xab, 0x0c, 0x38, 0xfe } }
-  gTpmDeviceInstanceTpm20PttPtpGuid       = { 0x93d66f66, 0x55da, 0x4f03, { 0x9b, 0x5f, 0x32, 0xcf, 0x9e, 0x54, 0x3b, 0x3a } }
\ No newline at end of file
+  gTpmDeviceInstanceTpm20PttPtpGuid       = { 0x93d66f66, 0x55da, 0x4f03, { 0x9b, 0x5f, 0x32, 0xcf, 0x9e, 0x54, 0x3b, 0x3a } }
```
Comment (miczyg1) [inline Vlv2TbltDevicePkg/PlatformPkg.dec]: @krystian-hebel could You clean it up please? `git add -p`...

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: Vlv2TbltDevicePkg/PlatformPkg.dec (hunk 1 above)
Comment (krystian-hebel) [reply Vlv2TbltDevicePkg/PlatformPkg.dec]: @miczyg1 should be fixed now. Editing files that don't follow any standards is painful...

**Review by miczyg1 — APPROVED** [review]

**Comment by miczyg1** [comment]
@pietrushnic are You ok with the changes?

**Review by pietrushnic — APPROVED** [review]

**Comment by miczyg1** [comment]
Closing PR, since everybody approved.

**Pull request closed without merging**